Libplanet.Action.Tests/Common/IncrementAction.cs

```diff
@@ -0,0 +1,37 @@
+using Bencodex.Types;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+
+namespace Libplanet.Action.Tests.Common
+{
+    public sealed class IncrementAction : IAction
+    {
+        public static readonly Address IncrementAddress =
+            new Address("0000000000000000000000000000000000000123");
+
+        public IncrementAction()
+        {
+        }
+
+        public IValue PlainValue => Bencodex.Types.Dictionary.Empty;
+
+        public void LoadPlainValue(IValue plainValue)
+        {
+        }
+
+        public IWorld Execute(IActionContext ctx)
+        {
+            IWorld states = ctx.PreviousState;
+            IAccount account = states.GetAccount(ReservedAddresses.LegacyAccount);
+            int value = 0;
+
+            if (account.GetState(IncrementAddress) is Integer integer)
+            {
+                value = (int)integer.Value + 1;
+            }
+
+            account = account.SetState(IncrementAddress, new Integer(value));
+            return states.SetAccount(ReservedAddresses.LegacyAccount, account);
+        }
```

Comment:
Is this test necessary?

Reply:
I created this for testing `BeginBlockActions` because there wasn't a suitable Action available. If there's a good Action for testing, I'll use that instead. For reference, `EndBlockActions` tests use `MinerReward`.

Reply:
Then, IMO name like `UpdateXXXAcion` is more fitting in this case. While `MinerReward` expresses well that mining rewards are added at the end of a block, `IncrementAction` does not seem to express that part well.

Reply:
[10b7d38](https://github.com/planetarium/libplanet/pull/3701/commits/10b7d3842326fc58cef706ac3623ab2ad6db50de#diff-63a1233cf4456bade4a58f6da6a9c7f37ec6bbdb7b664092781619c006376dd5) how about `UpdateValueAction`.